htdocs/modulebuilder/template/admin/about.php

```diff
@@ -23,18 +23,7 @@
  */
 
 // Load Dolibarr environment
-$res = 0;
```

Comment:
Please separate PR that change the code syntaxt from PR that has effect on architecture change.

Reply:
https://github.com/Dolibarr/dolibarr/pull/15915